File: Algoritmos Traduzidos/Perl/Algoritmos_Luis/60.pl
#Entrar com a razão de uma PA e o valor do 1º termo. Calcular e imprimir o 10º termo da série.

print "introduza o 1o termo\n";
$termo = <STDIN>;
print "introduza a razao\n";
$razao = <STDIN>;
$dec = $termo + (9*$razao);
print "O 10o termo desta progressao aritmetica e: $dec\n";

File: Algoritmos Traduzidos/Ruby/Algoritmos_Luis/60.rb
#Entrar com a razão de uma PA e o valor do 1º termo. Calcular e imprimir o 10º termo da série.
puts "introduza o 1o termo"
termo = gets
puts "introduza a razao"
razao = gets
dec = termo.to_f + (9*razao.to_f)
puts "O 10o termo desta progressao aritmetica e: #{dec}"